src/common/components/forms/OfferForm.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { FC, useEffect, useState } from 'react';
```

Comment:
Going to offer some more refactor recommendations for the repo but propose we keep this flat at `src/components/OfferForm`